packages/components/src/FilterBar/_docs/FilterBar.stories.tsx

```diff
@@ -91,105 +95,87 @@ export const Playground: StoryFn<typeof FilterBar> = () => {
-//   const { hideFilter } = useFilterBarContext()
-//   const label = "Curry"
-
-//   return (
-//     <FilterBarSelect
-//       label={label}
-//       renderTrigger={(triggerProps): JSX.Element => (
-//         <FilterButtonRemovable
-//           triggerButtonProps={{ ...triggerProps }}
-//           removeButtonProps={{ onClick: () => hideFilter(label) }}
-//         />
-//       )}
-//       items={[
-//         { label: "Lamb", value: "lamb" },
-//         { label: "Beef", value: "beef" },
-//       ]}
-//     />
-//   )
-// }
-
-// export const OldPlayground: StoryFn<typeof FilterBar> = () => {
-//   const [filtersState, setFiltersState] = useState({})
-
-//   return (
-//     <div>
-//       <FilterBar onChange={setFiltersState}>
-//         <FilterPancake
-//           label="Chocolate"
-//           renderTrigger={(triggerProps): JSX.Element => (
-//             <FilterButton {...triggerProps} />
-//           )}
-//         />
-
-//         <FilterDRP
-//           id="drp"
-//           locale="en-AU"
-//           label="Dates"
-//           renderTrigger={(triggerProps): JSX.Element => (
-//             <FilterButton {...triggerProps} />
-//           )}
-//         />
+// Component must be created to access the context.
+// Provider is in FilterBar, thus hook cannot be called in instantiating component.
+const VanillaPancake = (props: { id: string }): JSX.Element => {
+  const { setOpenFilter } = useFilterBarContext()
+  return (
+    <FilterPancake
+      id={props.id}
+      onChange={(): void => setOpenFilter("coffee")}
+    />
+  )
+}
 
-//         <VanillaPancake />
+export const Playground2: StoryFn<typeof FilterBar> = () => {
+  const [selectedValues, setSelectedValues] = useState<Record<string, any>>({})
+  const [filtersState, setFiltersState] = useState<StateWithoutComponent>({})
 
-//         <FilterBarSelect
-//           label="Coffee"
-//           renderTrigger={(triggerProps): JSX.Element => (
-//             <FilterButton {...triggerProps} />
-//           )}
-//           items={[
-//             { label: "Short black", value: "short-black" },
-//             { label: "Long black", value: "long-black" },
-//             { label: "Batch brew", value: "batch-brew" },
-//           ]}
-//         />
+  const filters: Filter[] = [
+    {
+      id: "chocolate",
+      label: "Chocolate",
+      Component: <FilterPancake id="chocolate" />,
+    },
+    {
+      id: "drp",
+      label: "Dates",
+      Component: <FilterDRP id="drp" locale="en-AU" />,
+    },
+    {
+      id: "vanilla",
+      label: "Vanilla",
+      Component: <VanillaPancake id="vanilla" />,
+      isRemovable: true,
+      isUsableWhen: state => state["chocolate"]?.selectedValue !== undefined,
+    },
+    {
+      id: "coffee",
+      label: "Coffee",
+      Component: <FilterPancake id="coffee" />,
+    },
+    {
+      id: "carrots",
+      label: "Carrots",
+      Component: <FilterPancake id="carrots" />,
+      isInitHidden: true,
```

Comment:
Perhaps we can operate under the assumption that all stateful props here are the "initial" ones, so we don't end up with `initialValue` `isInitHidden` etc.

Although, the `Component` could be responsible for `defaultValue` as part of their standard "form-like" API.